gamemodes/terrortown/gamemode/shared/sh_main.lua

```diff
@@ -182,6 +182,9 @@ hook.Add("PreRegisterSWEP", "TTT2RegisterSWEP", TTT2RegisterSWEP)
 -- @hook
 -- @realm shared
 function GM:TTT2Initialize()
+	-- Migrate all changes of TTT2
+	migrations.Apply()
```

Comment:
Maybe move this to an earlier place. Gotta see where it makes the most sense, but it should be early enough to catch new Convars and database calls.